xtas/core.py

```diff
@@ -68,8 +68,8 @@ def configure(config, import_error="raise", unknown_key="raise"):
             else:
                 logger.warn(msg)
 
-    app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
```

Comment:
Not one that I remember. I probably copied this habit from the [Celery docs example](http://docs.celeryproject.org/en/latest/reference/celery.html#celery.Celery.config_from_object).